python-lib/ner_utils_spacy.py

```diff
@@ -32,8 +32,7 @@ def get_spacy_model(language: str):
 def extract_entities(text_column, format: bool, language: str):
     # Tag sentences
     nlp = get_spacy_model(language=language)
-    docs = nlp.pipe(text_column.values)
-
+    docs = nlp.pipe(text_column.values, n_process=-1, disable=["tok2vec", "tagger", "parser", "attribute_ruler", "lemmatizer"], batch_size=100)
```

Comment:
Great job catching this. I think it would be cleaner & faster to [not even load](https://spacy.io/usage/processing-pipelines#disabling) them.
I think we can do
```python
nlp = spacy.load(language_model,  exclude=["tok2vec", "tagger", "parser", "attribute_ruler", "lemmatizer"])
```
in the try except part instead


Also was the default batch size slower for you? I think it's [1000](https://github.com/explosion/spaCy/blob/00e7bf5ffdd44eb327bbd85e03a00b3495221c94/spacy/default_config.cfg#L24) by default

Reply:
I was uncertain what is the typical text analyzed by the plugin, and since 1000 is a lot more than 100 it would mean suddenly vastly increasing its RAM usage. Because the chunksize needs to be roughly 2 * batch_size * cpu_count to stream about properly, that would mean suddenly going from 100 to 8000 sized chunks. 

Which is ok as long as the rows are not like whole books or stuff like that, so I'm a bit wary 🤔 

If you retry with bigger batches and chunks and get a lot better perf, it could make sense to up it, otherwise safe is good :-D  

Good point about not even loading them, it will make things indeed clearer 👍

Reply:
I changed the load() call. What I suggest is merging this PR (it targets your branch) if you're ok with the modifs and then proceed to merge your branch and release the plugin in a timely manner : since this PR also contains the version pin for sudachipy, we shouldn't make it hang.

Reply:
Nice! It goes from 5min50 prior to this PR to 1min50 with this PR on a 10K row dataset for me! Amazing job 👻